frontend/tet/admin/src/components/editor/PhoneInput.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import TetPosting from 'tet/admin/types/tetposting';
+import { useFormContext } from 'react-hook-form';
+import { PhoneInput as HdsPhoneInput } from 'hds-react';
+import Id from 'shared/types/id';
+import { RegisterOptions } from 'react-hook-form';
+
+// TODO add minusStepButtonAriaLabel and plusStepButtonAriaLabel
+type Props = {
+  id: Id<TetPosting>;
```

Comment:
`PhoneInput` is quite common component, so it would be nice to move instantly to `shared` folder so other projects could use it too. Only thing to change is the `TetPosting` to some generic like `T` then use it like `<PhoneInput<TetPosting> ... >`